validations/kyc.test.ts

```diff
@@ -58,6 +64,52 @@ describe('/kyc', () => {
       Object.values(KycStatus),
     )
   })
+  if (config.clientApiKey && config.providerId) {
+    it('sends kyc webhooks', async () => {
+      const beforeKycAdd = new Date().toISOString()
```

Comment:
seems like you may actually want to use IHL's `/clock` endpoint for this

Reply:
otherwise clock drift could cause you to fail the test

Reply:
I decided just to nix the time check in favor of address. I can add it back in if people feel strongly